src/calcium/qqbar.jl

```diff
@@ -1152,6 +1174,35 @@ function cospi(a::QQBarFieldElem)
    return z
 end
 
+@doc raw"""
+    sincospi(a::QQBarFieldElem)
```

Comment:
Should I really add a docstring here? The generic one for `sincospi` seems sufficient (as do those for `sinpi`, `cospi`. Although the examples maybe add some value?